src/main/scala/nl.knaw.dans.easy.bagstore/server/BagStoreServerComponent.scala

```diff
@@ -32,7 +32,7 @@ trait BagStoreServerComponent {
   class BagStoreServer(serverPort: Int) {
 
     private val server = new Server(serverPort) {
-      this.setHandler(new ServletContextHandler(ServletContextHandler.NO_SESSIONS) {
+      this.setHandler(new ServletContextHandler(ServletContextHandler.SESSIONS) {
```

Comment:
@jo-pol can you comment on whether this change is correct? With the former value (`NO_SESSIONS`) it didn't work, but on the other hand I don't want sessions; I just want password validation.
Am I taking the wrong approach for this?